tests 2/test_runtime_card_sanity.py

```
@@ -4800,6 +5815,919 @@ def test_v205_release_check_warns_on_blocked_pm25_normalization():
+    hass = SimpleNamespace(config=RejectPathAccess())
+    try:
+        asyncio.run(
+            services_mod.async_create_dashboard_for_entry(
+                hass,
+                entry,
+                layout="v2_mobile",
+                title="Humidity Intelligence",
+                url_path="../configuration",
+            )
+        )
+    except Exception as err:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)